yt/python/yt/local/bin/yt_local

```diff
@@ -225,6 +225,11 @@ def add_start_subparser(subparsers):
     parser.add_argument("--cell-tag", type=int, default=1,
                         help="cluster cell tag")
 
+    parser.add_argument("--job-proxy-logging-mode", choices=["sandbox", "separate_directory"], default="sandbox", 
+                        help="job proxy logging mode")
+    parser.add_argument("--job-proxy-logging-sharding-key-length", type=int, default=None,
+                        help="number of symbols for sharding key in separate_directory logging mode")
```

Comment:
I am not sure that this option should be represented here. Since I can't imagine the scenario in which it is useful.